src/lib/const.ts

```diff
@@ -337,29 +337,29 @@ export const SEPARATOR = `\n\n${color.yellow(
   `^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^`
 )}\n\n`;
 
-const MAIN_AQUA_FILE_STATUS_TEXT = `export status
-
--- example of running a service deployed using 'fluence deal deploy'
+const MAIN_AQUA_FILE_STATUS_TEXT = `
+-- example of running services deployed using 'fluence deal deploy'
 -- with worker '${DEFAULT_WORKER_NAME}' which has service 'MyService' with method 'greeting'
 
-func status():
-    workersInfo <- getWorkersInfo()
-    dealId = workersInfo.deals.${DEFAULT_WORKER_NAME}.dealId
-    print = (answer: string, peer: string):
-      Console.print([answer, peer])
+export runDeployedServices
 
-    answers: *string
+data Answer:
+    answer: string
+    peer: string
+
+func runDeployedServices() -> *Answer:
+    workersInfo <- getWorkersInfo()
+    dealId = workersInfo.deals!.defaultWorker!.dealId
+    answers: *Answer
     workers <- resolveSubnetwork(dealId)
     for w <- workers! par:
         on w.metadata.peer_id via w.metadata.relay_id:
             answer <- MyService.greeting("fluence")
-            answers <<- answer
-            print(answer, w.metadata.peer_id)
+            answers <<- Answer(answer=answer, peer=w.metadata.relay_id!)
 
-    Console.print("getting answers...")
```

Comment:
Why? I'd keep it

Reply:
I just think it's mostly redundant, but I can return it back if you think it's not